ci/pipelines/toolbox-staging-deploy.yml

```diff
@@ -60,13 +77,11 @@ jobs:
             source:
               repository: amazon/aws-cli
           params:
-            AWS_ACCESS_KEY_ID: ((readonly_access_key_id))
-            AWS_SECRET_ACCESS_KEY: ((readonly_secret_access_key))
-            AWS_SESSION_TOKEN: ((readonly_session_token))
-            AWS_ASSUME_ROLE_ARN: arn:aws:iam::((pay_aws_staging_account_id)):role/concourse
-            AWS_REGISTRY_ID: "((pay_aws_staging_account_id))"
+            AWS_ACCESS_KEY_ID: ((.:role.AWS_ACCESS_KEY_ID))
+            AWS_SECRET_ACCESS_KEY: ((.:role.AWS_SECRET_ACCESS_KEY))
+            AWS_SESSION_TOKEN: ((.:role.AWS_SESSION_TOKEN))
             AWS_REGION: eu-west-1
-            AWS_PAGER: ""
+            AWS_REGISTRY_ID: "((pay_aws_staging_account_id))"
             AWS_DEFAULT_OUTPUT: text
```

Comment:
suspect this isn't needed now

Reply:
It appears we actually need this otherwise we get the error here https://cd.gds-reliability.engineering/teams/pay-deploy/pipelines/toolbox-staging-deploy/jobs/toolbox-staging-deploy/builds/43

Reply:
ah ok, yeah makes sense